### ISSUE TITLE
Applicant - Bilingual Language Changes

### ISSUE THREAD

**Issue by joshdrink** [GCTC-NTGC/TalentCloud#826]
# Description
Text proposed by Lauren:

## English Essential
```
This position requires fluency in English in both written and verbal communication. As part of the assessment of your language abilities, the hiring manager may ask you to complete some assessment steps in English, such as interview questions or an exam.

You can submit this initial application in either official language of your choice (English or French).
```

## French Essential
```
This position requires fluency in French in both written and verbal communication. As part of the assessment of your language abilities, the hiring manager may ask you to complete some assessment steps in French, such as interview questions or an exam.

You can submit this initial application in either official language of your choice (English or French).
```

## Bilingual - Advanced
```
This position requires advanced fluency in both French and English. This means that you have strong reading, writing and verbal communication skills in both official languages, and can take on job duties in either French or English. As part of this selection process, your language abilities will be tested by the Public Service Commission of Canada (link).

You can complete all other steps of this assessment process in the official language of your choice, including the initial application, interview, exam and any other evaluation components.
```

**Comment by joshdrink** [comment]
As discussed, this issue also requires a migration to change "Bilingual" to "Bilingual - Advanced"

**Comment by gggrant** [comment]
Resolved by #824
